src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1723,7 +1731,7 @@ def forward(
 
     def generate(
         self,
-        inputs: Optional[torch.Tensor] = None,
```

Comment:
This was badly copied from the original `generate` function. Since we've added `generate` directly to Whisper here there is no need to call the general `generate` input: `"inputs"` => we can directly call it `input_features`. This creates a much nicer UX since a user can now do the following:

inputs = processor(audio)  # the feature extractor will create a dict with the name `input_features`
model.generate(**inputs)

`inputs` is deprecated here.